### PR TITLE
21 add gdpr question

### PR DESCRIPTION
**Pull request by LucaFumis**
Added paragraph for GDPR question to "before you begin" section.
If user selects 'yes', then 'What type of data are you submitting' section becomes enabled.
The 'yes, I agree part' could be a checkbox rather than a radio button, but currently I don't think we have one in the style lab.

### CODE REVIEW

**Comment by meladawy** [comment]
I was testing the function from my side. When you select "I Don't agree" I think you should block the user from "Save & Continue"...you can disable the "Save & continue" button if user doesn't agree on the GDPR notice

File: src/app/pages/submission-pages/overview-page/overview-page.component.html [hunk 1]
```diff
@@ -34,7 +34,22 @@ <h2 class="padding-top-xlarge">New data submission</h2>
...
-           You have selected : <span class="font-weight-bold">{{savedControlled == 1 ? 'Yes' : 'No'}}</span> <a (click)="onChangeField('controlled')"> (Change)</a>
+           You have selected : <span class="font-weight-bold">{{savedControlled === '1' ? 'Yes' : 'No'}}</span> <a (click)="onChangeField('controlled')"> (Change)</a>
+          </div>
+        </fieldset>
+
+
+        <fieldset class="large-form padding-bottom-large padding-top-large">
+          <legend>This service requires limited processing of your personal data in order to function.<br />By using the site you are agreeing to this as outlined in our <a href="https://www.ebi.ac.uk/data-protection/privacy-notice/ebi-wide-submission" target="_blank">Privacy Notice</a> and <a href="https://www.ebi.ac.uk/about/terms-of-use" target="_blank">Terms of Use</a></legend>
+          <!-- TODO: this should be a checkbox rather than radio button, but need to create styles for it match -->
+          <!-- <input type="checkbox" checked="savedGDPR" name="gdpr" value="1" formControlName="gdpr" id="gdprYes" (change)="onUpdateField('gdpr')">
```
Comment (meladawy) [inline src/app/pages/submission-pages/overview-page/overview-page.component.html]: Can you remove this code if its not gonna be used?

File: src/app/pages/submission-pages/overview-page/overview-page.component.html [hunk 2]
```diff
@@ -34,7 +34,22 @@ <h2 class="padding-top-xlarge">New data submission</h2>
...
+            <label for="gdprYes">Yes, I agree</label> -->
+          <input type="radio" *ngIf="!savedGDPR" name="gdpr" value="1" formControlName="gdpr" id="gdprYes" (change)="onUpdateField('gdpr')">
+            <label for="gdprYes" *ngIf="!savedGDPR">Yes, I agree</label>
```
Comment (meladawy) [inline src/app/pages/submission-pages/overview-page/overview-page.component.html]: Its better if you have label in the same indentation / alignment of input as its not wrapped by input.

File: src/app/pages/submission-pages/overview-page/overview-page.component.ts [hunk 3]
```diff
@@ -285,6 +288,10 @@ export class OverviewPageComponent implements OnInit {
       delete this.savedControlled;
     }
 
+    if(fieldName === 'gdpr'){
```
Comment (meladawy) [inline src/app/pages/submission-pages/overview-page/overview-page.component.ts]: Can you leave a space after if and before the curly bracket. Same for line 310

**Comment by LucaFumis** [comment]
Thanks Maged. Should be fixed now.

**Pull request merged**